examples/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+authors = [
+    "Tide Developers",
+]
+description = "Tide web server examples"
+documentation = "https://docs.rs/tide"
+keywords = ["tide", "http", "web", "framework", "async"]
+categories = [
+  "network-programming",
+  "asynchronous",
+  "web-programming::http-server"
+]
+edition = "2018"
+license = "MIT OR Apache-2.0"
+name = "tide-examples"
```

Comment:
Also, if this is not going to be published then for ease of running it would be nice to drop the `tide-` prefix here. Then users will be able to `cargo run -p examples --bin body_types` to run a specific example.

Reply:
And done :)